tests/postman/ElectionGuard API Tests.postman_collection.json

```diff
@@ -1052,15 +1103,280 @@
1\",\n        \"type\": \"school\"\n      },\n      {\n        \"contact_information\": {\n          \"address_line\": [\"1234 Pahk Avenue\", \"Harris, Ozark 99999\"],\n          \"email\": [\n            {\n              \"annotation\": \"inquiries\",\n              \"value\": \"inquiries@harris-township.gov\"\n            }\n          ],\n          \"name\": \"harris Town Hall\",\n          \"phone\": [{ \"annotation\": \"domestic\", \"value\": \"123-456-7890\" }]\n        },\n        \"name\": \"Harris Township\",\n        \"object_id\": \"harris-township\",\n        \"type\": \"township\"\n      }\n    ],\n    \"name\": {\n      \"text\": [\n        { \"language\": \"en\", \"value\": \"Hamiltion County General Election\" },\n        {\n          \"language\": \"es\",\n          \"value\": \"Elecci\\u00f3n general del condado de Hamilton\"\n        }\n      ]\n    },\n    \"parties\": [\n      {\n        \"abbreviation\": \"WHI\",\n        \"color\": \"AAAAAA\",\n        \"logo_uri\": \"http://some/path/to/whig.svg\",\n        \"name\": { \"text\": [{ \"language\": \"en\", \"value\": \"Whig Party\" }] },\n        \"object_id\": \"whig\"\n      },\n      {\n        \"abbreviation\": \"FED\",\n        \"color\": \"BBBBBB\",\n        \"logo_uri\": \"http://some/path/to/federalist.svg\",\n        \"name\": { \"text\": [{ \"language\": \"en\", \"value\": \"Federalist Party\" }] },\n        \"object_id\": \"federalist\"\n      },\n      {\n        \"abbreviation\": \"PPL\",\n        \"color\": \"CCCCCC\",\n        \"logo_uri\": \"http://some/path/to/people-s.svg\",\n        \"name\": { \"text\": [{ \"language\": \"en\", \"value\": \"People's Party\" }] },\n        \"object_id\": \"peoples\"\n      },\n      {\n        \"abbreviation\": \"LIB\",\n        \"color\": \"DDDDDD\",\n        \"logo_uri\": \"http://some/path/to/liberty.svg\",\n        \"name\": { \"text\": [{ \"language\": \"en\", \"value\": \"Liberty Party\" }] },\n        \"object_id\": \"liberty\"\n      },\n      {\n        \"abbreviation\": \"CONST\",\n        \"color\": \"EEEEEE\",\n        \"logo_uri\": \"http://some/path/to/democratic-repulbican.svg\",\n        \"name\": {\n          \"text\": [{ \"language\": \"en\", \"value\": \"Constitution Party\" }]\n        },\n        \"object_id\": \"constitution\"\n      },\n      {\n        \"abbreviation\": \"LBR\",\n        \"color\": \"FFFFFF\",\n        \"logo_uri\": \"http://some/path/to/laobr.svg\",\n        \"name\": { \"text\": [{ \"language\": \"en\", \"value\": \"Labor Party\" }] },\n        \"object_id\": \"labor\"\n      },\n      {\n        \"abbreviation\": \"IND\",\n        \"color\": \"000000\",\n        \"logo_uri\": \"http://some/path/to/independent.svg\",\n        \"name\": { \"text\": [{ \"language\": \"en\", \"value\": \"Independent\" }] },\n        \"object_id\": \"independent\"\n      }\n    ],\n    \"spec_version\": \"v0.95\",\n    \"start_date\": \"2020-03-01T08:00:00-05:00\",\n    \"type\": \"general\"\n  }\n}\n",
```

Comment:
This manifest is one that I actually simplified down to one contest with two ballots, but there's still a lot of cruft.  I'd like to come back and simplify it once I get everything working, but I'd prefer to worry about that later.

Reply:
is it possible to use external files in postman for things like this instead of having it embedded in?  That would make editing this easier and make the overall postman files smaller and more managable.

Reply:
Unfortunately, I could only find that option when running it in the runner, which would remove the ability to run individual steps.